### ISSUE TITLE
 Mis-posiitoned figure

### ISSUE THREAD

**Issue by barrypress** [arXiv/html_feedback#1281]
### Description

This is perhaps trivial; in the HTML conversion, Figure 1 has become mis-positioned at the start of the section rather than inline the text as in the source material.

### (Required) What is this issue most closely related to? Select one.

Figures

### Internal issue ID

1e1432d6-077c-46c0-8c14-a91aaa3bad72

### Paper URL

https://arxiv.org/html/1706.03762v7/

### Browser

Safari/605.1.15

### Device Type

Macbook

**Comment by dginev** [comment]
Right, this is in a way expected and not a "bug" per se. Figures and tables float in latex, to an appropriate placement for successful page layout. They float _less_ in the HTML conversion, as we do not have page constraints -- you will largely see them appear at the exact flow locations they were authored.
